lsst_efd_client/efd_helper.py

```diff
@@ -12,6 +16,42 @@
 from .efd_utils import merge_packed_time_series
 
 
+@dataclass
+class TopicField:
+    """Dataclass to hold schema information.  It also adds an `astropy.units.Unit`
+    if one can be coerced from the ``units`` field passed to the constructor.
+
+    Parameters
+    ==========
+    name : `str`
+        Name of the field
+    type : `str`
+        Type of the data in th efield: e.g. int, float
+    description : `str` (optional)
+        Description of the field.  Defaults to `None`
+    units : `str` (optional)
+        String specifying the  units for this field.  Defaults to `None`
+    aunits : `u.Unit`
+        This is derived from the value in ``units`` if possible
+    """
+    name: str  # Name of the field
+    type: str  # Type of the data in the field: e.g. int, float
```

Comment:
Notice that this is the Avro data type, it gets converted to an InfluxDB data type and to a Pandas data type in the end... perhaps just let the user use `Pandas.DataFrame.info()` to find the data types?

Reply:
Hmm.  So you are saying leave type information out completely?  It seems nice to have for debugging, i.e. to check if what is coming from influxDB is what is meant, but maybe we should make people query the schema registry by hand in that instance

Reply:
I left out the type information but kept the units because those are not pushed through influxDB.